### ISSUE TITLE
Accessibility: `error` in `Modifier.semantics` has no effect on macOS

### ISSUE THREAD

**Issue by ialokim** [JetBrains/compose-multiplatform#2277]
As the title states. See the following example code:

```kotlin
fun main() = singleWindowApplication {

    val (input, setInput) = remember { mutableStateOf("") }

    TextField(
        input,
        setInput,
        Modifier.semantics {
            if (input != "") error("Test")
        }
    )
}
```

EDIT: Writing something in the text field doesn't have any effect on macOS with VoiceOver. I would expect it to read out the error as soon as it is called in the `semantics` block or at least when focusing the respective TextField after it was called.

**Comment by hfhbd** [comment]
Did you add the import for `error`? Otherwise the stdlib error function is used, which crashes as expected your application.

```kotlin
import androidx.compose.material.TextField
import androidx.compose.runtime.remember
import androidx.compose.runtime.mutableStateOf
import androidx.compose.ui.Modifier
import androidx.compose.ui.semantics.semantics
import androidx.compose.ui.semantics.error
import androidx.compose.ui.window.singleWindowApplication

fun main() = singleWindowApplication {

    val (input, setInput) = remember { mutableStateOf("") }

    TextField(
        input,
        setInput,
        Modifier.semantics {
            if (input != "") error("Test")
        }
    )
}
```

BTW I don't see any visual change with this modifier on macOS with dev770.

**Comment by ialokim** [comment]
Yes, that was obviously it, my bad.

Still it doesn't have any effect on macOS with VoiceOver. I would expect it to read out the error as soon as it happens or at least when focusing the respective TextField.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2277">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
